src/platforms/browser/WebPlatform.mjs

```diff
@@ -171,16 +171,27 @@ export default class WebPlatform {
         /* WebGL blits automatically */
     }
 
-    registerKeyHandler(keyhandler) {
-        this._keyListener = e => {
+    registerKeydownHandler(keyhandler) {
+        this._keydownListener = e => {
```

Comment:
Not necessarily a style issue, but I'd prefer this arrow function to have () around arguments.